thenewboston_node/business_logic/tests/test_memory_blockchain/test_validations/test_blockchain_state_validations.py

```diff
@@ -17,8 +17,3 @@ def test_validate_account_root_files_raises(memory_blockchain: MemoryBlockchain,
     blockchain.blockchain_states = []
     with pytest.raises(ValidationError, match='Blockchain must contain at least one blockchain state'):
         blockchain.validate_blockchain_states()
-
-
-@pytest.mark.skip('Not implemented yet')
-def test_validate_account_root_file_balances():
-    raise NotImplementedError
```

Comment:
Tested in `thenewboston_node/business_logic/tests/test_blockchain_base/test_validate_root_account_file_balances.py`